libs/fundamental-redirects/index.js

```diff
@@ -92,6 +96,21 @@ const LOCALE_PATTERNS = [
     },
     { permanent: true }
   ),
+  // Retired locales
+  redirect(
+    new RegExp(
+      `^(?<locale>${Array.from(RETIRED_LOCALES.keys()).join(
+        "|"
+      )})(/(?<suffix>.*)|$)`,
+      "i"
+    ),
+    ({ locale, suffix }) => {
+      const join = suffix && suffix.includes("?") ? "&" : "?";
+      return `/en-US/${
```

Comment:
I wonder, can you do 
```
const { DEFAULT_LOCALE } = require('../constants');
...
```
plus
```suggestion
      return `/${DEFAULT_LOCALE}/${
```

Reply:
@peterbe Will do!